src/config.cpp

```diff
@@ -123,11 +126,21 @@ static config_type configuration;
 static config_type internal;
 static config_type internal_bak;
 
-bool budget::load_config(){
+bool budget::load_config() {
     if(!load_configuration(path_to_home_file(".budgetrc"), configuration)){
         return false;
     }
 
+    fs::path config_home;
+    if(auto xdg_config_home = std::getenv("XDG_CONFIG_HOME")) {
+        config_home = fs::path{xdg_config_home};
+    } else {
+        config_home = fs::path{home_folder()} / ".config";
+    }
+    if(!load_configuration((config_home / "budget" / "budgetrc").string(), configuration)) {
+        return false;
+    }
```

Comment:
The problem with this function is that it can load the configuration from two different files at once, which is not expected. 

Could you make a `config_file_path` function in the same idea as `home_folder `and then use it here to choose the file to open?

Reply:
sure, I thought it wouldn't hurt to read both

Reply:
We could handle this case properly, but I do not think there is much added value in parsing two configuration files. So I think it's better to load a single one and look at the path based on XDG conventions.